web/styles/compose.css

```diff
@@ -151,6 +151,10 @@
 .autocomplete_secondary {
     opacity: 0.8;
     font-size: 85%;
+
+    & * {
+        display: inline-block;
+    }
```

Comment:
Let's just specify where we want this property to apply. I don't like using `*` here.

Reply:
It is used for `<code>` element which I have specified explicitly in css.